pokemon-app/src/components/PokemonCry.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect, useRef } from "react";
+import pokeCries from "../utils/pokeCries";
+import reactoadSound from "../assets/sounds/reactoad.mp3";
+
+const PokemonCry = ({ pokeName }) => {
+  // Set the initial state for rendering the false.
+  const [isPokemonsCries, setIsPokemonsCries] = useState(false);
+  // Set the sound url to an empty string
+  const [pokeSoundUrl, setPokeSoundUrl] = useState("");
+  // Initialize the audio reference.
+  const audioRef = useRef();
+
+  // First useEffect sets the rendering to true, and sets the sound url
+  // which depends on the pokemon's name.
+  useEffect(() => {
+    // If either names exist, continue.
+    if (pokeName === "Reactoad" || pokeCries[pokeName]) {
```

Comment:
reactoad pokemon? :0

Reply:
I love you and miss you 😘 @dannysanchez559